### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by Lukeghenco**
Adding more accurate ES6 styling for functions, fixed the reducer action that wasn't setting the state back to 0 correctly and cleaned up some odd spacing and unneeded semicolons. @aturkewi @PeterBell

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,27 +22,30 @@ Next, let's define a reducer function in `reducers/counterReducer.js` to pass to
...
   switch (action.type) {
     case 'INCREMENT_COUNT':
-      return state + 1;
+      return state + 1
     case 'DECREMENT_COUNT':
-      return state - 1;
+      return state - 1
     case 'RESET_COUNT':
-      return 0;
+      state = 0
```
Comment (aturkewi) [inline README.md]: Why set `state = 0` here? This shouldn't have to be here I don't think. Just returning 0 should modify the state in the store like in all the other cases in this switch

File: README.md [hunk 2]
```diff
@@ -49,19 +54,19 @@ Let's start our server by using `npm start`. Visit http://localhost:8080 in your
 import { createStore } from 'redux';
 import counterReducer from './reducers/counterReducer';
```
Comment (aturkewi) [inline README.md:55]: This section is to 'play around in console' but you cannot use `import` in the console. It seems we want to tell people to add these `console.log()` and `store.dispatch()` calls to `index.js` instead and refresh the page while watching the console.

**Review by aturkewi — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (Lukeghenco) [reply README.md:55]: good catch. I'll update the readme for this one.

Comment (Lukeghenco) [reply README.md:55]: Finished

**Comment by Lukeghenco** [comment]
@aturkewi changes have been made and ready for merge

**Pull request merged**